### PR TITLE
cleanup, text notifications added

### PR DESCRIPTION
**Pull request by charles-cowart**
redundant filepaths in plate input cell removed.
text notifications for user from MacKenzie added.

### CODE REVIEW

File: notebooks/amplicon_pre_prep_file_generator.ipynb [hunk 1]
```diff
@@ -21,6 +21,14 @@
    "source": [
     "# Knight Lab Amplicon Mapping File (Pre-Preparation File) Generator\n",
     "\n",
+    "<font color='red'><B>A VERY IMPORTANT Note on Plate Compression and Positions:</B></font>\n",
+    "\n",
+    "This notebook works SPECIFICALLY with the STANDARD EpMotion compression format ONLY.\n",
+    "PRIMERS are tied to our standard SPECIFIC PLATE POSITIONS outlined in the code, and CANNOT be changed.\n",
+    "\n",
+    "If you are not compressing in the standard compression format (position 1, position 2, position 3, position 4) that uses the plate map template below,\n",
+    "DO NOT USE THIS NOTEBOOK TO GENERATE THE MAPPING FILE.\n",
```
Comment (antgonza) [inline notebooks/amplicon_pre_prep_file_generator.ipynb:30]: mapping file?? is this the pre-preparation file?

Comment (mmbryant23) [reply notebooks/amplicon_pre_prep_file_generator.ipynb:30]: Good catch, Antonio. I am still using old language.
Charlie - Can you please change "mapping file" to "pre-preparation file" in lines 30? Thank you.

**Comment by mmbryant23** [comment]
Thanks, Charlie - After we change "mapping file" to "pre-preparation file" in line 30 (my mistake), things will be good to go. I can send you a meeting ID and we can quickly go over updating the notebook through terminal, if that's ok with you.

File: notebooks/amplicon_pre_prep_file_generator.ipynb (hunk 1 above)
Comment (charles-cowart) [reply notebooks/amplicon_pre_prep_file_generator.ipynb]: @mmbryant23 line changed, but just I noticed mapping file is used in several other places in the notebook as well - sometimes pre-prep is in parentheses but sometimes not. Should we go ahead and remove mapping-file entirely?

**Comment by charles-cowart** [comment]
@mmbryant23 sure! you can send me a link whenever you're ready.

**Pull request merged**